packages/cli/test/unit/import-lb3-model/migrate-model.test.js

```diff
@@ -33,7 +33,6 @@ describe('importLb3ModelDefinition', () => {
       expect(toJSON(modelData.settings)).to.deepEqual({
         // By default, LB3 models are not strict
         strict: false,
-        replaceOnPUT: true,
```

Comment:
FYI: `replaceOnPUT` is a flag set by juggler, it's used for backwards compatibility with LoopBack 2.x. 

Quoting from https://loopback.io/doc/en/lb4/Model.html:

> `replaceOnPUT` - This entry is no longer supported as LB4 controllers scaffolded by LB4 controller, PUT is always calling replaceById. Users are free to change the generated code to call patchById if needed.